DependencyInjection/DoctrineExtension.php

```diff
@@ -168,13 +162,13 @@ protected function loadDbalConnection($name, array $connection, ContainerBuilder
         }
 
         // event manager
-        $container->setDefinition(sprintf('doctrine.dbal.%s_connection.event_manager', $name), new $definitionClassname('doctrine.dbal.connection.event_manager'));
+        $container->setDefinition(sprintf('doctrine.dbal.%s_connection.event_manager', $name), new ChildDefinition('doctrine.dbal.connection.event_manager'));
```

Comment:
Should we take this as an opportunity to make things easier to review?
```suggestion
        $container->setDefinition(
            sprintf('doctrine.dbal.%s_connection.event_manager', $name), 
            new ChildDefinition('doctrine.dbal.connection.event_manager')
        );
```